### PR TITLE
[DOC-2126] restructure bulk ops page

### PR DESCRIPTION
**Pull request by ashleykwtam**
Address DOC-2126:
- Changed the bulk ops page to only include backups and restores 
- Moved import / export examples to their statement pages
- Changed URL links to incorporate new page names

Preview:
- [Customer-Owned Backups](https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/cockroachcloud/take-and-restore-customer-owned-backups.html)
- [Managed-Service Backups](https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/cockroachcloud/use-managed-service-backups.html)
- [IMPORT](https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/stable/import.html)
- [EXPORT](https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/stable/export.html)

### CODE REVIEW

**Comment by kathancox** [comment]
@ashleykwtam I haven't looked through the PR much just commenting here on the titling as requested!

We'll want to confirm the titling with Liv, but I think as placeholders so that the PR can be reviewed perhaps we want to go with something like the following:

Restore Data from a Backup --> Use Managed-Service Backups
My thinking for keeping this more generic with "Use" is that there will be more features in the Cloud Console shortly beyond just restoring a managed-service backup. 

Run Bulk Operations --> Take and Restore Customer-Owned Backups
I'm not wholly convinced with "customer-owned" here, but since that is now the feature name, it probably makes sense to have it in the title.

Again, Liv will probably have more ideas here—though let's incorporate these ^ titles and change up the links and nav for so we can get a good view of what everything looks like.

Once you feel ready, you can tag me for an initial review, or let me know if you want me to take a look at anything else in draft view.

File: _includes/cockroachcloud/userfile-examples/backup-userfile.md
Comment (kathancox) [inline _includes/cockroachcloud/userfile-examples/backup-userfile.md:47]: On the customer-owned backups file, I have made a comment about how we could split this up into the two sections Back up data and Restore data

File: _includes/sidebar-data-cockroachcloud.json
Comment (kathancox) [inline _includes/sidebar-data-cockroachcloud.json:254]: Also need to add the changes in page to the `_redirects` file (https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#file-naming), so that anyone who tries to get to the former Run Bulk Operations page will be redirected to the Take and Restore Customer-Owned Backups.

**Review by kathancox — CHANGES_REQUESTED** [review]
Hey Ashley, Thanks for working on this, and particularly for moving the import/export examples from the page. This'll be much clearer. I have left some comments — overall everything looks good. I think now it'd be great if you could rework the intros to the two renamed pages as it seems like we have collected lots of different sentences over the last several months that need reorganizing for the new titles.

Once you've worked through that, I can take another quick look before Liv's review.

File: cockroachcloud/take-and-restore-customer-owned-backups.md
Comment (ashleykwtam) [reply cockroachcloud/take-and-restore-customer-owned-backups.md:60]: removing this sentence as there is already a statement describing the examples in the 'Examples' section

'The examples on this page demonstrate how to take customer-owned backups'

File: cockroachcloud/use-managed-service-backups.md
Comment (kathancox) [reply cockroachcloud/use-managed-service-backups.md:23]: I can take a look before you include in a commit, if that's what you prefer! Feel free to send me the shots directly

**Review by nickvigilante — APPROVED** [review]
Approved `_redirects` file only.

File: cockroachcloud/use-managed-service-backups.md
Comment (kathancox) [reply cockroachcloud/use-managed-service-backups.md:29]: Your dedicated screenshot made me double check the UI, it looks like **Status** doesn't exist in that UI any longer, so that can be deleted. Might be good to include a screenshot showing incrementals and full backups for dedicated: 

image

Feel free to use that one ^

File: cockroachcloud/use-managed-service-backups.md
Comment (kathancox) [inline cockroachcloud/use-managed-service-backups.md:34]: In the dedicated section, perhaps change the sentence to include incremental backups as well: 

"A list of your full cluster backups displays on your cluster's **Backups** page." 

This is a remnant from when full backups were only available.

**Review by kathancox — CHANGES_REQUESTED** [review]
Hey Ashley, 

Thanks for making those initial changes. Can you request a review from Liv now? It would be great to see if Liv has feedback on the titling and pages generally.

Once you have incorporated any feedback Liv has, you can request a review from me again to finalize the PR before merging.

Thanks again, this is going to really help define the Cloud backup types once merged!

**Review by livlobo — COMMENTED** [review]
On this page https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/cockroachcloud/take-and-restore-customer-owned-backups.html

We describe Backup and Restore -- but only the basics. However, they can essentially support more functionality  i.e. encrypted backups, scheduled backups, locality aware backups, etc. 

I think it might make more sense to organize by FEature first and then by product type.
@ashleykwtam @kathancox 

Backup & Restore Data 
> Overview
> Architecture
> Customer-owned backups
>> Customer-owned backups on CRDB Cloud clsuters
>> Customer-owned backups on CRDB SH clusters       
> Managed-service backups 
>> Managed-service backups on CRDB Cloud clusters 
> Full & Incremental backups
>> On Cloud (anything different than SH?)
>> On Cloud (anything different than SH?)
> Backups with Revision history & PITR 
>> On Cloud (anything different than SH?)
>> On Cloud (anything different than SH?)
...

**Comment by kathancox** [comment]
> On this page https://deploy-preview-15808--cockroachdb-docs.netlify.app/docs/cockroachcloud/take-and-restore-customer-owned-backups.html
> 
> We describe Backup and Restore -- but only the basics. However, they can essentially support more functionality i.e. encrypted backups, scheduled backups, locality aware backups, etc.
> 
> I think it might make more sense to organize by FEature first and then by product type. @ashleykwtam @kathancox
> 
> image

@livlobo @ashleykwtam I think it is a good idea for us all to restructure the nav according to product type intersected with feature. However, I think we should discuss this in more depth (possible iterations) and Ashley merge this PR as a "first fix" and then we can iterate. The main reason is that the docs for v23.1 will be cut on Feb 7, anything we don't merge will have to be copied over... And that will make way more work on this PR. So, I'm going to open an issue, set it highest priority to be completed in the Feb sprint to rearrange according to what we decide. https://cockroachlabs.atlassian.net/browse/DOC-6675

**Review by kathancox — APPROVED** [review]
LGTM Ashley, the other reorg of backup/restore nav will be separate work from this PR. Feel free to merge when conflicts are resolved. Lmk if you have questions! thanks

**Pull request merged**
